### ISSUE TITLE
Post source code for Accessibility_Toolbar.dll

### ISSUE THREAD

**Issue by compuguy** [ThePacielloGroup/WebAccessibilityToolbar#98]
The software posted here is licenced under the GPL Version 3, but there is no source to the Accessibility_Toolbar.dll, which may be a violation of the GPL. Could the source code be posted to this dll?

**Comment by matatk** [comment]
You're right and we're aware of this -- we are currently trying to set up a build system that enables us to keep all the source on GitHub, so that we will be in full compliance (development is gradually moving to GitHub from being on our own servers only).  The same issue affected the Colour Contrast Analyser and aViewer, and we have ongoing "issues" to track the progress towards the full build system (e.g. ThePacielloGroup/aviewer#3) and the source for aViewer and the Analyser is now on GitHub, though we still need to create the build scripts.

Getting the source code for the Web Accessibility Toolbar into GitHub is the last task before completing the build system.  We are a small company, so unfortunately sometimes these sorts of changes can take time; our intent was to make the software available so that people could use it as soon as possible, and follow up as quickly as possible with the full code and build system — which we're now much closer to completing, as those two projects on which this one depends now have full code in GitHub.

In terms of timetable: I doubt we will be able to do this within the next couple of months, though we will update this issue as we make progress towards it.  If you require access to the source before then, I can make enquiries on your behalf, though I do not personally have access to it, and this too may take some time.  Sorry for the inconvenience at the moment -- we are looking forward to completing this transition to GitHub particularly as it will enable us to much more easily make releases.
